packages/engine/src/avatar/systems/AvatarCameraInputSystem.ts

```diff
@@ -192,6 +196,6 @@ const execute = () => {
 
 export const AvatarCameraInputSystem = defineSystem({
   uuid: 'ee.engine.AvatarCameraInputSystem',
-  insert: { with: InputSystemGroup },
+  insert: { after: InputSystemGroup },
```

Comment:
why has this been changed to after?

Reply:
on a call we were trying to get the AvatarCameraInputSystem capture of input to happen last, it did not work, will revert